app/views/comments/_comment_date.erb

```diff
@@ -4,4 +4,10 @@
       <%= decorated_comment.readable_publish_date %>
     </time>
   </a>
-</div>
+  <% if !decorated_comment.edited_at.blank? %>
+    •
+    <span title="<%= decorated_comment.edited_at.strftime('%A, %d %B %Y, %T') %>">
```

Comment:
Great! The only thing I can think of is that the previous date (creation date) is going to be localized by JavaScript, this isn't as it's not a time element. 

What if we change it to: `• edited on [SHORT DATE]` and use `<time>` like in the other date?

cc @ludwiczakpawel

Reply:
I don't really understand what you mean by the previous date being localized

Reply:
Sure, sorry for the unclear explanation.

Basically, if you look at 

https://github.com/forem/forem/blob/87590e77d3d2ec74acd0d843a904f90cec1c86b8/app/views/comments/_comment_date.erb#L3-L5

you'll notice how the date is wrapped into a `<time>` HTML element. This, combined with some JS magic on the frontend, will localize the hover date (basically convert it to whatever language the user has set their browser to). For example this is how I see the comment date if I set my browser in Italian:

<img width="276" alt="Screenshot 2020-10-02 at 12 35 01 PM" src="https://user-images.githubusercontent.com/146201/94914685-c40c6f00-04ab-11eb-9029-fabd114e1044.png">

and this is if I set it back to English:

<img width="306" alt="Screenshot 2020-10-02 at 12 35 20 PM" src="https://user-images.githubusercontent.com/146201/94914719-d4bce500-04ab-11eb-9fcc-0307eacd666c.png">

This is accomplished by the semantic `<time>` element and https://github.com/forem/forem/blob/master/app/assets/javascripts/initializers/initializeCommentDate.js

Unfortunately though this can't be easily accomplished in this case as the date is rendered only server side. 

My suggestion to change the working to `edited on <time>...</time>` was to be able to display a short date to the user but also be able to localize the full date.

UX wise is not entirely clear that one is suppose to over on the `EDITED` as well, since it's not a link unlike the creation date which points to the full page. 

Anyway, let's wait for the designers feedback on this.

Let me know if anything isn't still clear, it's a lot of info :D

Reply:
I think the UI is good as it is: so only "Edited" indicator and the actual timestamp in the tooltip.

FY: I have a separate branch where I'm currently working on comments UI update  - so I'll take this indicator for another ride then. For now, this is good as is :) Great job!

Reply:
Yippie. Thank you. I'm so happy to be helping out